snmp/datadog_checks/snmp/data/default_profiles/raritan-dominion.yaml

```diff
@@ -0,0 +1,62 @@
+extends:
+  - _base.yaml
+  - _generic-if.yaml
+
+metadata:
+  device:
+    fields:
+      vendor:
+        value: "raritan"
+
+sysobjectid:
+  - 1.3.6.1.4.1.13742.*
+  - 1.3.6.1.4.1.13742.3.2.10
+metrics:
+  - MIB: RemoteKVMDevice-MIB
+    symbol:
+      name: memory.usage
+      OID: 1.3.6.1.4.1.13742.3.1.1.0
+  - MIB: RemoteKVMDevice-MIB
+    symbol:
+      name: cpu.usage
+      OID: 1.3.6.1.4.1.13742.3.1.2.0
+  - MIB: RemoteKVMDevice-MIB
+    table:
+      name: systemPowerSupplyTable
+      OID: 1.3.6.1.4.1.13742.3.1.3
+    symbols:
+      - name: raritan.dominion.systemPowerSupply
```

Comment:
I think maybe `raritan.remotekvm` could be a better prefix than `raritan.dominion`.

`remotekvm` seems more stable since it's related to the MIB (RemoteKVMDevice-MIB)

Reply:
- `dominion` seems to be the product name.
- `remotekvm ` is based on the MIB name

and usually, MIB names are more stable than a product name (products name can change over time, but it's less that case for MIB names)